### ISSUE TITLE
suricate-server should take a configuration file as parameter

### ISSUE THREAD

**Issue by marco-buttu** [marco-buttu/suricate#64]
We want:

```
suricate-server -f path_of_file.yaml
suricate-server # withoud -f it reads the default configuration

```

**Comment by marco-buttu** [comment]
This should be a suricate-server command, because is the server awares of the configuration file. But the server is a service, and it only know where to take one configuration file. If you want to have a temporary configuration file just move the current one  (*config.yaml*) and use your temporary file intead.

**Comment by marco-buttu** [comment]
Fixed in commit 4c69b72.
